### DISCUSSION TITLE
v2在react项目里使用过程中部分样式无效

### DISCUSSION THREAD

**Discussion by huaxiangsirenbutong** [alibaba/formily#2324] [Q&A 2.x] [answered]
npx create-react-app antd-demo-ts --template typescript 快速创建的react项目
问题展示：
![image](https://user-images.githubusercontent.com/37055241/137849990-cd5ada8e-f688-4260-a966-252eedf20292.png)
相关依赖版本：
![image](https://user-images.githubusercontent.com/37055241/137850236-e8ffcbe6-30b4-455c-a88a-84859c344dc9.png)
求问怎么解决

**Comment by janryWang** [comment, marked as answer]
你要手动引入样式呀，看看官网的codesandbox例子

**Reply by huaxiangsirenbutong** [reply]
好的
